app/views/landmarks/show.html.erb

```diff
@@ -1,9 +1,9 @@
 <h3><%= @landmark.name %></h3>
-<%= @landmark.user.email %>
+Creator: <%= @landmark.user.email %>
 <%= image_tag("#{@landmark.image}", height: '250', width: '250') %>
```

Comment:
@mlg- sure, I can make a note of this in our task on styling, though if we remove it now then we'll have jumbo images making the site unusable. Thoughts?